src/main/java/com/cloudbees/jenkins/plugins/gcloudsdk/GCloudInstallation.java

```diff
@@ -0,0 +1,65 @@
+package com.cloudbees.jenkins.plugins.gcloudsdk;
+
+import hudson.EnvVars;
+import hudson.Extension;
+import hudson.model.EnvironmentSpecific;
+import hudson.model.Node;
+import hudson.model.TaskListener;
+import hudson.slaves.NodeSpecific;
+import hudson.tools.ToolDescriptor;
+import hudson.tools.ToolInstallation;
+import hudson.tools.ToolProperty;
+import jenkins.model.Jenkins;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * @author <a href="mailto:nicolas.deloof@gmail.com">Nicolas De Loof</a>
+ */
+public class GCloudInstallation extends ToolInstallation implements NodeSpecific<GCloudInstallation>, EnvironmentSpecific<GCloudInstallation> {
+
+    @DataBoundConstructor
+    public GCloudInstallation(String name, String home, List<? extends ToolProperty<?>> properties) {
+        super(name, home, properties);
+    }
+
+
+    public void buildEnvVars(EnvVars env) {
+        env.override("PATH+GCLOUD", getHome()+"/bin");
```

Comment:
[CustomTool](https://github.com/jenkinsci/customtools-plugin/blob/master/src/main/java/com/cloudbees/jenkins/plugins/customtools/CustomTool.java) does not implement buildEnvVar, so how does this compare ?

Also, based on JDK sample this seems not to be necessary